Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -195,6 +195,9 @@ public void Roslyn_Settings_ValidSetup_NonLegacyServer_MergeSettings()
         {
             // Arrange
 
+            var dir = TestUtils.CreateTestSpecificFolder(TestContext);
```

Comment:
The QP-generated ruleset now needs to exist for the test to pass, since the product code is now reading it (previously, it just wrote the file path it was given to the merged ruleset file)